aws/resource_aws_iam_server_certificate_test.go

```diff
@@ -87,28 +93,35 @@ func TestAccAWSIAMServerCertificate_disappears(t *testing.T) {
 }
 
 func TestAccAWSIAMServerCertificate_file(t *testing.T) {
-	var cert iam.ServerCertificate
+	var _cert iam.ServerCertificate
```

Comment:
I use `_` here to denote that I don't actually care about `_cert` in this test; loading the file (with and without windows line endings) and applying should be sufficient, because follow up `plan` as part of the test will tell us if there's a diff due to endings.